ms_pro/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "ms_pro",
```

Comment:
Please change name of directory and fontend project to "frontend" or "user-interface". Suppose we name NLP module like this module. How can I understand which module is frontend and which module is NLP?
Git convention for commits is in ordering. This commit is better to be "Initialize frontend module". Please do same its later